capstone.html

```diff
@@ -0,0 +1,1062 @@
+			<p class="mb-5">
+				Suppose you use a digital input pin on the Arduino board, but it’s not connected to anything. Is the state of the pin HIGH or LOW? 
+				For it to be HIGH, the pin would need to be connected to the VCC power source. For the state to be LOW, the pin would need to be 
+				connected to GND. When the state is indeterminate like this, the pin is said to have a <b>floating signal</b>, which we should avoid.
+				<br><br>
+				The solution is what we call a <b>pull-up resistor</b>. The input pin is connected to a high-value resistor R1 whose other terminal 
+				is attached to VCC (as depicted below).
+				<br><br>
+				The intent is to set the input pin to the HIGH voltage. We often use a 10kΩ resistor to do so because it drives the current down low 
+				(recall V=IR). Now, the input pin has an effective resistance in the range of 100 MΩ. That’s right: mega-ohms. 
+			</p>
+			<div class="mb-5">
+				<img class="pj-img-fluid-end" src="./assets/images/project_5/ops_project_5_pull_up_explanation.png" width = "350">
+				<p class="mb-3">
+					The resistance of the pull-up resistor R1 and effective resistance of the input pin divide the voltage VCC such that the voltage 
+					at the input pin is equal to VCC (HIGH).
+				</p>
+				<p class="mb-3">
+					Now, let’s introduce the button: one terminal attached to the input pin and the other GND.
+				</p>
+				<p class="mb-3">
+					When the button is pressed, the voltage at the input pin is shorted to GND (LOW).
+				</p>
+			</div>
+			<p class="mb-3">
+				With this circuit, the input pin will either be VCC or GND (HIGH or LOW).
+				<br><br>
+				When you use a button or, in our case, the <span class="text-green">SW</span> pin of the joystick, you should use a pull-up resistor. 
+				Instead of building an external circuit, you can use an <b>internal pull-up resistor</b> in the Arduino board. You may do so by setting 
+				the pin mode as follows:
+			</p>
+			<p class="mb-3">
+				<code class="pj-code language-arduino">pinMode(pin, INPUT_PULLUP);</code>
+			</p>
+			<p class="mb-5">
+				There are also <b>pull-down resistors</b>, which we will not discuss here.
+			</p>
+			
+			<h2 class="mb-3 fw-bold" id="pj-l293d-quadruple-half-h-driver">L293D Quadruple Half-H Driver</h2>
+			<p class="mb-5">
+				We will use an L293D chip to drive the two DC motors of the rover. The H-bridge should be familiar to you. This time, the L293D will do 
+				the heavy lifting. Within this IC are two H-bridges, making it possible to drive two motors bidirectionally.
+			</p>
+
+			<h4 class="mb-3 fw-light">Reading the Big Scary Manual</h4>
+			<p class="mb-3">
+				This time around, we will not thoroughly discuss the driver’s operation. You will have to read the datasheet, and figure it out yourself.
+				<br><br>
+				<a class="link-primary" href="https://drive.google.com/file/d/1XgJbA4hapVwiDeIkIh0e_JCzlX8x-nO8/view">L293D Quadruple Half-H Driver Datasheet.pdf</a>
```

Comment:
I believe that we want these PDFs to be actually downloaded and linked directly.

Check out `/assets/datasheets`

Reply:
Fixed

Reply:
looks good